src/CommandHandler.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exan\Fenrir;
+
+use Exan\Fenrir\Command\FiredCommand;
+use Exan\Fenrir\Const\Events;
+use Exan\Fenrir\Parts\ApplicationCommand;
+use Exan\Fenrir\Rest\Helpers\Command\CommandBuilder;
+use Exan\Fenrir\Websocket\Events\InteractionCreate;
+use Exan\Fenrir\Websocket\Events\Ready;
+
+class CommandHandler
+{
+    private bool $activated = false;
+
+    private array $commands = [];
+
+    private bool $devMode = false;
```

Comment:
Why is called `devMode` if it seems to represent whether the command is attached to a guild or not?

Reply:
Discords docs recommend using guild commands instead of global commands for testing due to caching. For this reason `registerCommand` method is dynamic in what kind of commands it creates. When in dev mode, its a guild command, when not in dev mode, its a global command.

Intended use can be seen here: https://github.com/dc-Ragnarok/Lyngvi/commit/a89f1979916fe2dbf8f45fae6cc0019a97adeb49

If a `DEV_GUILD_ID` is set in the `.env`, it passes it to the command builder, which then uses guild commands. To switch to global commands for production without having to make code changes, you just don't set said variable.

Reply:
Ahh, that's a fun fact I never knew